### PR TITLE
Forms 7414

### PR DESCRIPTION
**Pull request by barshat7**
## Description

- Adding Email Input & Telephone Input Custom Components.
- Some minor fixes of names and OOTB items for some core components

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/wizard/.content.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
-    jcr:description="Adaptive Form Wizard (v1) component"
+    jcr:description="Wizard - a multi-step forms showing one step at a time"
```
Comment (nit23uec) [inline src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/wizard/.content.xml]: is this string reviewed by CHL? seems like a grammatical mistake.

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/tabsontop/_cq_template.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
     jcr:primaryType="nt:unstructured"
-    jcr:title="Tabs on top"
+    jcr:title="Horizontal Tabs"
```
Comment (nit23uec) [inline src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/tabsontop/_cq_template.xml:4]: we should also include two panels by default in tabs just like in accordion.

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/wizard/.content.xml (hunk 1 above)
Comment (barshat7) [reply src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/wizard/.content.xml]: reverted to earlier version

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/tabsontop/_cq_template.xml (hunk 2 above)
Comment (barshat7) [reply src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/tabsontop/_cq_template.xml:4]: Done

**Review by nit23uec — APPROVED** [review]

**Review by rismehta — APPROVED** [review]

**Pull request merged**